plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -19,9 +20,17 @@ import org.gradle.api.Plugin
 import org.gradle.api.Project
 import org.gradle.api.Task
 import org.gradle.api.plugins.ExtraPropertiesExtension
+import org.gradle.util.GradleVersion
 
 class SentryPlugin : Plugin<Project> {
     override fun apply(project: Project) {
+        Sentry.init {
+            it.dsn = "https://1053864c67cc410aa1ffc9701bd6f93d@o447951.ingest.sentry.io/5428559"
+            // read release dynamically somehow
```

Comment:
For the future: this could be done with
https://github.com/gmazzo/gradle-buildconfig-plugin

Reply:
indeed, good idea, we use it on https://github.com/getsentry/sentry-java as well